<launch>
    <arg name="param_file" default="$(find villa_yolocloud)/config/parameters.yaml" />
    <rosparam command="load" file="$(arg param_file)"/>

    <node pkg="villa_yolocloud" type="context_cloud_node" name="conttext_node" output="screen" respawn="true"></node>
</launch> 
